chapters/remote/remote-flows.md

```diff
@@ -0,0 +1,71 @@
+# Remote Flows with JinaD
+
+### Prerequisites
+
+Before you start, make sure you have read the [prerequisites for using JinaD](https://docs.jina.ai/chapters/remote/jinad.html#prerequisites).
+
+## REST API
+
+[Refer the detailed API reference for JinaD](https://api.jina.ai/daemon/#tag/flows)
+
+A common use case is running the Flow on the remote with pods distributed across different machines.
+
+#### Examples
+
+Define `flow.yml` with the correct `JinaD` host details.
+
+```yaml
+jtype: Flow
+pods:
+    - uses: _logforward
+      host: 1.2.3.4:8000
+      upload-files:
+        - custom_encoder_1.yml
+    - uses: _logforward
+      host: 2.3.4.5:8000
```

Comment:
also what is this other IP, another machine?

maybe add a section above the YAML to say:
given that:
- 1.2.3.4 is machine 1
- 2.3.4.5 is machine 2
- 8000 is the port of JinaD

Reply:
Fixed